gcloud-java-storage/src/main/java/com/google/gcloud/storage/Bucket.java

```diff
@@ -179,8 +179,8 @@ public Blob get(String blob, BlobSourceOption... options) {
    * @throws StorageException upon failure
    */
   Blob create(String blob, byte[] content, BlobTargetOption... options) {
```

Comment:
Unrelated to this PR.

content-type is a required field (and a common one to set).
Do you think we should change this signature to:
`Blob create(String blob, byte[] content, String contentType, BlobTrargetOptions... options)`?

If so, javadoc can see that if contentType is null a default value will be picked. (Maybe we could reference the value from a constant in the code...)